pkg/artifacts/artifact_storage_test.go

```diff
@@ -285,3 +285,41 @@ func TestGetArtifactStorageWithoutConfigMap(t *testing.T) {
 		t.Fatalf("want %v, but got %v", expectedArtifactPVC, pvc)
 	}
 }
+
+func TestGetArtifactStorageWithPvcConfigMap(t *testing.T) {
+	logger := logtesting.TestLogger(t)
+	prName := "pipelineruntest"
+	for _, c := range []struct {
+		desc                    string
+		configMap               *corev1.ConfigMap
+		expectedArtifactStorage ArtifactStorageInterface
+	}{{
+		desc: "valid bucket",
+		configMap: &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: system.GetNamespace(),
+				Name:      PvcConfigName,
+			},
+			Data: map[string]string{
```

Comment:
We don't really test that the pvc size that will be requested is the right size (10Gi), we only check that no error is triggered when the config map is loaded. Perhaps we could "mock" the `getPVCSpec` method and verify that it's invoked with the expected values?

Reply:
added test for size